hw/ip/aes/data/aes.hjson

```diff
@@ -497,7 +497,19 @@
           The AES unit is not stalled (0) or stalled (1) because there is previous
           output data that must be read by the processor before the AES unit can
           overwrite this data.
-          This flag not meaningful if MANUAL_OPERATION=1 (see Control Register).
+          This flag is not meaningful if MANUAL_OPERATION=1 (see Control Register).
+        '''
+        tags: ["excl:CsrNonInitTests:CsrExclCheck"]
+      }
+      { bits: "5",
```

Comment:
@rasmus-madsen I actually wanted to change the bit order in the status register as I felt this one is related to the stall bit. Changing the bit order currently is not possible (UVM DV fails because of hardcoded bit positions). But I noticed that this is being reworked by your PR #4810. Once it is merged, I will rebase this PR and change the bit order.

Reply:
Thanks @rasmus-madsen for your review. A struct would be nice. Do you know if we can automate that with our regtool? Otherwise we might just move the problems to the interface between RTL and SW. Alternatively, we could see if DV can be made to use the offset and mask values used by software to assemble the struct. For example, the auto-generated `aes_regs.h` currently contains:
```
// Control Register. Can only be updated when the AES unit is idle. If the
#define AES_CTRL_SHADOWED_REG_OFFSET 0x74
#define AES_CTRL_SHADOWED_OPERATION_BIT 0
#define AES_CTRL_SHADOWED_MODE_MASK 0x3f
#define AES_CTRL_SHADOWED_MODE_OFFSET 1
#define AES_CTRL_SHADOWED_MODE_FIELD \
  ((bitfield_field32_t) { .mask = AES_CTRL_SHADOWED_MODE_MASK, .index = AES_CTRL_SHADOWED_MODE_OFFSET })
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_ECB 0x1
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_CBC 0x2
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_CFB 0x4
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_OFB 0x8
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_CTR 0x10
#define AES_CTRL_SHADOWED_MODE_VALUE_AES_NONE 0x20
#define AES_CTRL_SHADOWED_KEY_LEN_MASK 0x7
#define AES_CTRL_SHADOWED_KEY_LEN_OFFSET 7
#define AES_CTRL_SHADOWED_KEY_LEN_FIELD \
  ((bitfield_field32_t) { .mask = AES_CTRL_SHADOWED_KEY_LEN_MASK, .index = AES_CTRL_SHADOWED_KEY_LEN_OFFSET })
#define AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_128 0x1
#define AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_192 0x2
#define AES_CTRL_SHADOWED_KEY_LEN_VALUE_AES_256 0x4
#define AES_CTRL_SHADOWED_MANUAL_OPERATION_BIT 10
#define AES_CTRL_SHADOWED_FORCE_ZERO_MASKS_BIT 11

// Status Register
#define AES_STATUS_REG_OFFSET 0x7c
#define AES_STATUS_IDLE_BIT 0
#define AES_STATUS_STALL_BIT 1
#define AES_STATUS_OUTPUT_VALID_BIT 2
#define AES_STATUS_INPUT_READY_BIT 3
#define AES_STATUS_ALERT_RECOVERABLE_BIT 4
#define AES_STATUS_ALERT_FATAL_BIT 5
```

Maybe some of these values could also be exported to DV.

Reply:
I don't know our reg tool well enough.
but It seem like a logical addition.

in have worked with RDL tools before that would provide a struct